request.go

```diff
@@ -148,3 +232,51 @@ func (c *Client) WaitTillProvisioned(path string) (err error) {
 	}
 	return
 }
+
+type RequestSelector func(Request) bool
+
+func IsFinished(r Request) bool {
+	switch r.Metadata.RequestStatus.Metadata.Status {
+	case "QUEUED", "RUNNING":
+		return false
+	}
+	return true
+}
+
+// WaitTillRequestsFinished will wait until list requests with applied filters does not
```

Comment:
Weird wording. How about "will wait until there are no more unfinished requests matching the given filter"?

Reply:
You marked this as resolved. Did you change anything?

Reply:
this github is weird. Clicked the wrong button as it seems.

Reply:
I wanted to avoid the impression that the filtering is done by the client.